config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
   resources :events, only: :create
   resources :integrations, only: %i(index show)
+
+  if Rails.env.development?
+    resources :events, only: :index
+  end
```

Comment:
It's good to have a reference of how events are supposed to look:

![screen shot 2015-09-18 at 3 23 11 pm](https://cloud.githubusercontent.com/assets/9178278/9972435/3648a08c-5e19-11e5-9989-9a133c1904db.png)